styles/scss/components/body-header.scss

```diff
@@ -0,0 +1,81 @@
+@import '../reset';
+@import '../variables';
+@import '../fonts';
+
+.body-header {
```

Comment:
jak możesz to  style zapisuj za pomocą camelCase bodyHeader wtedy nasz kod będzie bardziej spójny i czytelny z innymi branchami :)